### PR TITLE
New parser for Åland

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue
The current parser is only returning one datapoint at a time and don't include low values of export/import at all due to the nature of the source.
Fixes: #4235
## Description

This PR fetches the data from a iFrame at https://kraftnat.ax/driftcentralen/ and then parse that. The  benefit is that we no longer need to use OCR and it fetches the data for the last 24 hours making the whole parser and data more reliable for Åland.

>**Note**
>I am deliberately using a new file name so this will be easier to review, if it get's approved I will rename the file and archive the old parser.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
I will rename the parser file to `AX.py` once there is an approving review.

File: parsers/AX_NEW.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+                "fossil": float(fossil),
+                "gustavs": float(gustavs),
+                "wind": float(wind),
+                "consumption": float(consumption),
+            }
+        )
+    return data_list
+
+
+def formated_data(
```
Comment (unitrium) [inline parsers/AX_NEW.py:67]: This should probably live in the shared lib folder at some point.
It's probably ok for now to have it here, but I feel that all parsers would benefit from sharing such a function.

**Review by unitrium — APPROVED** [review]
Looks good!
Just a small thought on the format function that could be shared.

File: parsers/AX_NEW.py (hunk 1 above)
Comment (VIKTORVAV99) [reply parsers/AX_NEW.py:67]: This function in particular probably won't work well shared but we could make a shared function that formats data to our different return types (these could also include some basic type checks if we want).

**Comment by VIKTORVAV99** [comment]
Thanks, I'll do the final cleanup and archive the old parser before merging this.

**Pull request merged**